src/configurators/LayerConfigurator.js

```diff
@@ -1,6 +1,6 @@
 import ol from 'openlayers'
 
-import GroupLayer from '../layers/GroupLayer'
+import {GroupLayer} from '../layers/GroupLayer'
```

Comment:
See https://github.com/feross/standard/issues/609

space-in-brackets was removed with ESLint 1.0.